README.md

```diff
@@ -146,6 +146,18 @@ colcon build --packages-select zenoh_bridge_dds --cmake-args -DCMAKE_BUILD_TYPE=
 ```
 The `rosdep` command will automatically install *Rust* and *clang* as build dependencies.
 
+If you want to cross-compile the package on x86 device for any target, you can use the following command:
+```bash
+rosdep install --from-paths . --ignore-src -r -y
+colcon build --packages-select zenoh_bridge_dds --cmake-args -DCMAKE_BUILD_TYPE=Release  --cmake-args -DCROSS_ARCH=<target>
+```
+where `<target>` is the target architecture (e.g. `aarch64-unknown-linux-gnu`). The architechture list can be found [here](https://doc.rust-lang.org/nightly/rustc/platform-support.html).
+
+The cross-compilation uses `zig` as a linker. You can install it with:
```

Comment:
Please, update instructions according to [discussion](https://github.com/eclipse-zenoh/zenoh-plugin-dds/pull/147#issuecomment-1697036587)

Reply:
Done. @JEnoch